### ISSUE TITLE
How do I use ChainFury with SSL?

### ISSUE THREAD

**Issue by jan-koch** [NimbleBoxAI/ChainFury#123]
Hey,
sorry for this newbie question... I set up the Docker container on a VPS and have created a Let's Encrypt certificate. 

Accessing the web interface with HTTP works perfectly but with HTTPS, I get an error that the site cannot provide a secure connection. It says: "domain.com sent an invalid response"

Is there a setting I need to update in the config of ChainFury or is this a hosting-related (Hetzner) issue?

**Comment by jan-koch** [comment]
Got it to work, here's what I did:

I used Certbot to create SSL certificates using Let's Encrypt.

Copied all PEM files into a folder in the web root. Made those available in the Dockerfile and loaded Uvicorn with those files:

`# Add SSL
COPY ./cert cert
RUN update-ca-certificates

EXPOSE 8000
CMD ["uvicorn", "app:app", "--host", "0.0.0.0", "--port", "8000", "--workers", "1", "--ssl-keyfile", "./cert/privkey1.pem", "--ssl-certfile", "./cert/cert1.pem"]
`

I'm not an expert, what did I miss?
